integration-cli/docker_cli_run_test.go

```diff
@@ -61,6 +61,14 @@ func (s *DockerSuite) TestRunWithoutMemoryswapLimit(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestRunWithSwappiness(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "--memory-swappiness", "0", "busybox", "true")
```

Comment:
I was too fast regarding the code. will do some more testing to arrive at the consensus